crates/rolldown/tests/rolldown/function/extend/entry-wrapped-cjs-named/artifacts.snap

```diff
@@ -0,0 +1,99 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Assets
+
+## main.js
+
+```js
+
+
+//#region main.js
+var require_main = __commonJS({ "main.js"() {
+	require_main();
+} });
+
+//#endregion
+export default require_main();
+
+```
+---
+
+Variant: (format: Cjs)
+
+# Assets
+
+## main.js
+
+```js
+
+
+//#region main.js
+var require_main = __commonJS({ "main.js"() {
+	require_main();
+} });
+
+//#endregion
+module.exports = require_main();
+
+```
+---
+
+Variant: (exports: Named, extend: true, format: Iife, name: "module")
+
+# Assets
+
+## main.js
+
+```js
+(function(exports) {
+
+Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
Seems wrong. why does not cjs format emit this?

Reply:
Fixed it.